alerting/pagerduty.go

```diff
@@ -0,0 +1,71 @@
+package alerting
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/datajet-io/peekaboo/services"
+)
+
+const apiEndpoint = "https://events.pagerduty.com/generic/2010-04-15/create_event.json"
+
+//Pagerduty represents the configuration of the Pageruty account used for notifications
+type Pagerduty struct {
```

Comment:
it is not just a configuration, it looks more like a client, doesn't it? lets call it smth like PagerDutyClient